cli/cli.ts

```diff
@@ -484,11 +484,10 @@ function ciAsync() {
                     if (uploadDocs)
                         p = p
                             .then(() => buildWebStringsAsync())
-                            .then(() => crowdin.execCrowdinAsync("upload", "built/webstrings.json"));
+                            .then(() => crowdin.execCrowdinAsync("upload", "built/webstrings.json"))
+                            .then(() => crowdin.execCrowdinAsync("upload", "built/skillmap-strings.json"));
```

Comment:
skills maps are target specific; they should be uploaded under the arcade folder in crowdin.